### DISCUSSION TITLE
oap9.7  can't find virtual database

### DISCUSSION THREAD

**Discussion by ms787277296** [apache/skywalking#11858] [Q&A]
i use oap9.7, all microservice requests are normal. But in service menu, i can't find virtual database submenu. I check the agent which is directly downloaded from (https://www.apache.org/dyn/closer.cgi/skywalking/java-agent/9.1.0/apache-skywalking-java-agent-9.1.0-src.tgz), and the microservice  in did use mysql. I wonder why I can't find the virtual database submenu

**Comment by wu-sheng** [comment]
It is in the marketplace. Once you have virtual service detected, the menu shows up automatically.

**Reply by ms787277296** [reply]
sorry，What does marketplace mean

**Reply by wu-sheng** [reply]
The first menu at left top.

**Comment by ms787277296** [comment]
![image](https://github.com/apache/skywalking/assets/29354555/576c6178-c4ea-49cf-a74e-ae2dcf76ee80)
we use this config in our project, but oap seems fail to detect mysql. 
![image](https://github.com/apache/skywalking/assets/29354555/37a92495-c6e7-4522-ac8e-64428ed8f8d2)
the correct chain is below:
![image](https://github.com/apache/skywalking/assets/29354555/2cba60a7-5a94-4e0d-9398-614907a2bd93)
can you give me a suggestion. Thanks a lot

**Reply by wu-sheng** [reply]
Duplicate https://github.com/apache/skywalking/issues/11872
